### PR TITLE
remove LWT reviewer tools

### LINKED ISSUE #10755: Remove LWT reviewer tools
All pending LWT have been cleared from the reviewer tools, and all approved LWT have been converted to static themes so we can remove the special LWT reviewer tools.

### PR DESCRIPTION
**Pull request by eviljeff**
fixes #10755

### CODE REVIEW

File: src/olympia/addons/indexers.py [hunk 1]
```diff
@@ -330,8 +330,7 @@ def extract_document(cls, obj):
             # add-ons stored in ES.
             data['platforms'] = [amo.PLATFORM_ALL.id]
             try:
-                data['has_theme_rereview'] = (
-                    obj.persona.rereviewqueuetheme_set.exists())
+                data['has_theme_rereview'] = False
```
Comment (diox) [inline src/olympia/addons/indexers.py]: You can remove that entirely (and the associated checks in `test_indexers.py`)

File: src/olympia/reviewers/models.py [hunk 2]
```diff
@@ -1146,81 +1146,6 @@ def get_content_review_queue(cls, admin_reviewer=False):
...
-    def get_queryset(self):
-        qs = super(RereviewQueueThemeManager, self).get_queryset()
-        if self.include_deleted:
-            return qs
-        else:
-            return qs.exclude(theme__addon__status=amo.STATUS_DELETED)
-
-
-@python_2_unicode_compatible
-class RereviewQueueTheme(ModelBase):
```
Comment (diox) [inline src/olympia/reviewers/models.py]: Need an issue to remove those tables from the db entirely later

File: src/olympia/addons/indexers.py (hunk 1 above)
Comment (eviljeff) [reply src/olympia/addons/indexers.py]: I wasn't sure if it was used externally (though it's not documented, like none of the LWT stuff is).  I'll remove it.

Comment (eviljeff) [reply src/olympia/addons/indexers.py]: https://github.com/mozilla/addons-server/pull/10759/commits/e03f87ba6942fff0d4f09fd1efd639811a4de04d

File: src/olympia/reviewers/models.py (hunk 2 above)
Comment (eviljeff) [reply src/olympia/reviewers/models.py]: https://github.com/mozilla/addons-server/issues/10760

**Comment by wagnerand** [comment]
There is an issue with the LWT reviewer tools that prevents us from clearing the queues. I will follow up on Monday.

**Pull request merged**